src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -12,6 +14,49 @@ class AuthApplicationSection extends React.Component {
     let content;
     const verifiedAccountType = 3;// verified ID.me accounts are type 3
     const isVerifiedUser = this.props.userProfile.accountType === verifiedAccountType;
+    const savedForms = this.props.userProfile.savedForms;
+    const hasSavedForms = !!savedForms;
+    
+    //TODO:  (ceh) verify edu and burial form codes
+    const formTitles = {
```

Comment:
This feels like it should be broken down into two different helper objects and relocated to a helper file, but I didn't want to set up a file for just two helpers that aren't shared. Thanks for other thoughts on this.